### ISSUE TITLE
Spaces in files or directories fails when calling torrenttomedia

### ISSUE THREAD

**Issue by Jaerin** [clinton-hall/nzbToMedia#692]
So it appears that the argument handling of torrenttomedia isn't able to recognize spaces in file and directories.

This is the command line that I called manually from the command line:
D:\Programs\nzbToMedia>TorrentToMedia.py "D:\Torrent\Seeding\tv\" "Parks and Recreation S07E04 720p WEB-DL DD5.1 H.264.mkv" "tv" "639FC900038D3B407209D111A0F2D3E24337EA2F"

http://pastebin.com/X3iiYSwe

Seems that it is not keeping the quoted argument as a whole item.  Looks like from what I can tell it uses the same argument handling for nzbtomedia.

**Comment by clinton-hall** [comment]
this definately works ok in Linux.

About to test in Windows... I notice your have a `>` in your path above? is this a typo, or is that what you passed?
Notice in the log there is a single `"` in between the directory and the first word of the name... did you have a miss-balance of quotes?

**Comment by clinton-hall** [comment]
wow... confirmed in windows!!!

**Comment by clinton-hall** [comment]
Ah... so:
`D:\Programs\nzbToMedia\TorrentToMedia.py "D:\Torrent\Seeding\tv\" "Parks and Recreation S07E04 720p WEB-DL DD5.1 H.264.mkv" "tv" "639FC900038D3B407209D111A0F2D3E24337EA2F"`
will fail as per your log... but
`D:\Programs\nzbToMedia\TorrentToMedia.py "D:\Torrent\Seeding\tv" "Parks and Recreation S07E04 720p WEB-DL DD5.1 H.264.mkv" "tv" "639FC900038D3B407209D111A0F2D3E24337EA2F"`
works fine.

The difference is the directory we pass.. in the first 1 the directory ends in `\` which being an "escape" character in python (and posix... in fact everything except windows) this actually escapes the `"` after the directory and so the split of the rest of the args gets out of sync...

so simply removing the trailing `\` works fine for a manual run.

Do you get this same issue when the script is called from your torrent client?

**Comment by clinton-hall** [comment]
Any update here? was the trailing `\` able to be removed?

**Comment by Jaerin** [comment]
This can be closed.  Turns out it was a problem in the vbs that I was using to call the torrenttomedia.py script.

Had to put everything in quotes.  I manually attempted that run in the logs from the command line so the extra trailing slash on the directory was my fault.  Honestly wish there was a way in utorrent to like right-click rerun post-process to execute the command again.  Good luck getting them to implement anything users want though.
